Sources/Jobs/JobInstance.swift

```diff
@@ -68,10 +79,16 @@ struct JobInstanceData<Parameters: Codable & Sendable>: Codable {
     let parameters: Parameters
     /// Date job was queued
     let queuedAt: Date
+    /// When to execute a job in the future
+    let delayUntil: Date?
+    /// Number of attempts so far
+    let attempts: Int
```

Comment:
We need to look into this, because the non-optional field `attempts` will definitely be a breaking change.